packages/embark-utils/src/accountParser.js

```diff
@@ -1,3 +1,4 @@
+import { __ } from 'i18n';
```

Comment:
shouldn't this be `embark-i18n`?

Reply:
No `embark-i18n` actually doesn't provide `__`. That comes from `i18n` itself (confusing, I know!). This import is to ensure typesafety (would work without it as well)

Reply:
Oh, shoot, sorry. What does it provide?

Reply:
It configures and sets up `i18n` with a default language and exposes that configuration API so it can be used in tests.